src/tasks/portuguese/enem_challenge.py

```
@@ -0,0 +1,107 @@
+            labels = ["A", "B", "C", "D", "E"][: len(texts)]
+
+        return labels, texts
+
+    def _extract_expected_index(self, raw_sample: Dict, labels: list) -> int:
+        """Extract the expected answer index."""
+        answer_key = raw_sample.get("answerKey", raw_sample.get("answer", raw_sample.get("label")))
+        if isinstance(answer_key, int):
+            return answer_key
+        elif isinstance(answer_key, str) and answer_key in labels:
+            return labels.index(answer_key)
+        logger.warning(f"Missing answer key for sample, defaulting to 0")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)